demos/imagenet_classification_demo/infer_request_callback.hpp

```diff
@@ -0,0 +1,103 @@
+#include <cstdio>
+#include <string>
+#include <functional>
+#include <atomic>
+
+#include <inference_engine.hpp>
+
+#include <ie_iextension.h>
+
+void resizeImage(cv::Mat& image, int modelInputResolution) {
+    double scale = static_cast<double>(modelInputResolution) / std::min(image.cols, image.rows);
+    cv::resize(image, image, cv::Size(), scale, scale);
+
+    cv::Rect imgROI;
+    if (image.cols >= image.rows) {
+        int fromWidth = image.cols/2 - modelInputResolution/2;
+        imgROI = cv::Rect(fromWidth,
+                          0,
+                          std::min(modelInputResolution, image.cols - fromWidth),
+                          modelInputResolution);
+    }
+    else {
```

Comment:
Please, merge this line with previous one.

Reply:
Done

Reply:
No, it's not.

Reply:
Have not pushed to this repo yet, sorry.
I'll tell you after I finish.